### ISSUE TITLE
Radio button not working properly with ReactiveForms

### ISSUE THREAD

**Issue by meeroslav** [valor-software/ngx-bootstrap#1768]
**Description:**
When radio button is used with ReactiveForms instead of template driven forms it preserves active state.

**Expected behavior:**
When value of control changes (upon click) active state should change. Button should not have a local copy of input value.

**Existing behavior:**
Button creates local copy of value upon first click, and stays in the active state even when some other button had changed the value.

**Demo:**
[https://embed.plnkr.co/gfzYSQi5Ke1Kj9oag7gX/](https://embed.plnkr.co/gfzYSQi5Ke1Kj9oag7gX/)

**Comment by pantonis** [comment]
I can verify the same problem

**Comment by wgrabowski** [comment]
You can make it work by binding [value] input like in this example: https://embed.plnkr.co/uiTMW6/

**Comment by meeroslav** [comment]
@wgrabowski Not sure what should your plunkr prove, since it doesn't work as you can see from screenshot below and has the same code as my initial plunkr.

![image](https://cloud.githubusercontent.com/assets/881612/26401019/15a0732e-4084-11e7-9063-e875d8a6ddd3.png)

I think you've sent the wrong link perhaps, or forgot to hit save.

**Comment by meeroslav** [comment]
Additionally, I would expect ReactiveForms to work as designed by angular team, not forced to work with hacks. But, yes, hack is still better than nothing.

Currently unfortunately I have to do it manually like:

```html
<div class="btn-group">
  <label class="btn btn-primary" [ngClass]="{active: myForm.controls.myControl.value == 'Left'}">
    <input type="radio" formControlName="myControl" class="custom-control-input" value="Left">
    Left
  </label>
  <label class="btn btn-secondary" [ngClass]="{active: myForm.controls.myControl.value == 'Middle'}">
    <input type="radio" formControlName="myControl" class="custom-control-input" value="Middle">
    Middle
  </label>
  <label class="btn btn-secondary" [ngClass]="{active: myForm.controls.myControl.value == 'Right'}">
    <input type="radio" formControlName="myControl" class="custom-control-input" value="Right">
    Right
  </label>
</div>
```

**Comment by wgrabowski** [comment]
@meeroslav You're right my plunkr link was wrong. What I was trying to show is exactly what you've posted above: binding "value" property allows you to work around this problem.

**Comment by valorkin** [comment]
fix merged, will be released as v1.7.2
